docker/.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:18.12.1
+
+WORKDIR /cuttle/
+
+ENV NPM_CONFIG_LOGLEVEL=notice
+
+RUN apt-get update
```

Comment:
It's good practice, btw, to collapse these RUN lines where possible; this saves space by reducing the number of layers

```Dockerfile
RUN apt-get-update && \
         apt-get clean
```

Reply:
@ollien yeah totally, I also wanted to make sure we're running on npm 9.8 if possible to stop our package lock from jumping between versions 2 and 3.

I'll make that change here but I still want another pass eventually to pull in more files so it makes a new build when things change in root files, and to reduce the number of layers as you said.

Thanks for the feedback!